src/components/Book.jsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
Opps 🤦 , let's stick to best practice and project guidelines.

Let's remove this line, we don't need it so far we are passing the method down to the Book component as a prop.

In milestone 2, you were introduced to the `component` and `container` folders.

And it was stated that the `book.jsx` would be a presentational component.

Note:  Presentational components for stateless components and a container/page folder for other components that use the state/redux-store.

More Explanation: 
React bindings for Redux separate presentational components from container components. This approach can make your app easier to understand and allow you to more easily reuse components. In your components directory (presentational components) you have components that are not connected to the Redux store, and on your containers directory, you have components that are connected to the Redux store. Read More... https://www.pluralsight.com/guides/how-to-structure-redux-components-and-containers